fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIOCatsConversions.scala

```diff
@@ -55,7 +56,10 @@ trait BIOCatsConversions7 extends BIOCatsConversions8 {
 trait BIOCatsConversions8 extends BIOCatsConversions9 {
   @inline implicit final def BIOAsyncToAsync[F[+_, +_]](implicit F: BIOAsync[F]): cats.effect.Async[F[Throwable, ?]] with S9 = new BIOCatsAsync[F](F)
 }
-trait BIOCatsConversions9 {
+trait BIOCatsConversions9 extends BIOCatsConversions10 {
+  @inline implicit final def BIOParallelToParallel[F[+_, +_]](implicit F: BIOParallel[F]): cats.Parallel[F[Throwable, ?]] = new BIOCatsParallel[F](F)
```

Comment:
@neko-kai The conversion test would not work unless I added this. 
Is this correct?

Reply:
@Adriani277 Yep.